next-steps.md

```diff
@@ -0,0 +1,117 @@
+
+## Who can publish to a collection?
+
+Some collections are open to all contributors, while more curated collections are only accepting contributions from internal to Microsoft employees and specific partners.
+
+## What is a collection?
+
+A collection is an entry point to discover, browse and initialize an [Azure Developer CLI](https://github.com/Azure/azure-dev) template. Examples of collection are
+
+- [Awesome AZD](https://azure.github.io/awesome-azd/)
+- [AI Apps Gallery](#)
+
+and any other published list, meeting the criteria above
+
+## Step 1: Creating a new template from scratch
+
+If you are creating a new template from scratch, we recommend you use our scaffolding mechanism.
+
+To create a new template with all the necessary content, configuration and structure to successfully pass validation and qualify for curated collections, an author must use the scaffolding mechanism provided by executing  
+
+`azd init –t [starter-template-name]` 
+
+Starter templates available today are 
+
+Starter template name | URL | Use-case
+-- | -- | --
+ai-starter | [Azure-Samples/azd-ai-starter: Creates an Azure AI Service and deploys the specified models. (github.com)](https://github.com/azure-samples/azd-ai-starter) | AI without AI-Studio
+ai-studio-starter | [Azure-Samples/azd-aistudio-starter: Creates an Azure AI Studio hub, project and required dependent resources including Azure Open AI Service, Cognitive Search and more. (github.com)](https://github.com/Azure-Samples/azd-aistudio-starter)  | AI with AI-Studio
+
+### Initializing the template on your existing sample
+
+You can also initialize the template for an existing code base. To do so, follow [this link](https://learn.microsoft.com/azure/developer/azure-developer-cli/start-with-app-code)
+
+## Step 2: Build your application
+
+This stage maps to template status: `Template in progress`: the template has been generated, published to a repo, and the application is being developed. 
+
+While in progress, the template team can adopt the following development guidelines:
+
+Guideline type | Link
+-- | --
+Development Guidelines | [azd-template-artifacts/docs/development-guidelines/development-process.md at main · Azure-Samples/azd-template-artifacts (github.com)](https://github.com/Azure-Samples/azd-template-artifacts/blob/main/docs/development-guidelines/development-process.md) 
+Operational Guidelines | [azd-template-artifacts/docs/development-guidelines/operational-guidelines.md at main · Azure-Samples/azd-template-artifacts (github.com)](https://github.com/Azure-Samples/azd-template-artifacts/blob/main/docs/development-guidelines/operational-guidelines.md) 
+Best practices per domain | [azd-template-artifacts/docs/development-guidelines/best-practices-per-domain.md at main · Azure-Samples/azd-template-artifacts (github.com)](https://github.com/Azure-Samples/azd-template-artifacts/blob/main/docs/development-guidelines/best-practices-per-domain.md) 
```

Comment:
```suggestion
Recommended practices per domain | [azd-template-artifacts/docs/development-guidelines/recommended-practices-per-domain.md at main · Azure-Samples/azd-template-artifacts (github.com)](https://github.com/Azure-Samples/azd-template-artifacts/blob/main/docs/development-guidelines/best-practices-per-domain.md) 
```

I much prefer `recommended` over `best` for guidance since `best practices` can imply there’s only one solution or set of solutions and `recommended practices` is literally what _we_ are recommending in this context. :)